2025-04-18 20:44:26,838 INFO    Thread-2 (_run_job):68003 [wandb_setup.py:_flush():67] Current SDK version is 0.19.7
2025-04-18 20:44:26,839 INFO    Thread-2 (_run_job):68003 [wandb_setup.py:_flush():67] Configure stats pid to 68003
2025-04-18 20:44:26,839 INFO    Thread-2 (_run_job):68003 [wandb_setup.py:_flush():67] Loading settings from /Users/rudra_sarkar/.config/wandb/settings
2025-04-18 20:44:26,839 INFO    Thread-2 (_run_job):68003 [wandb_setup.py:_flush():67] Loading settings from /Users/rudra_sarkar/Documents/PhD IIT Madras/Intro to Deep Learning M Khapra DA6401/da6401 assignment 2/wandb/settings
2025-04-18 20:44:26,839 INFO    Thread-2 (_run_job):68003 [wandb_setup.py:_flush():67] Loading settings from environment variables
2025-04-18 20:44:26,839 INFO    Thread-2 (_run_job):68003 [wandb_init.py:setup_run_log_directory():647] Logging user logs to /Users/rudra_sarkar/Documents/PhD IIT Madras/Intro to Deep Learning M Khapra DA6401/da6401 assignment 2/wandb/run-20250418_204426-24f1so8s/logs/debug.log
2025-04-18 20:44:26,839 INFO    Thread-2 (_run_job):68003 [wandb_init.py:setup_run_log_directory():648] Logging internal logs to /Users/rudra_sarkar/Documents/PhD IIT Madras/Intro to Deep Learning M Khapra DA6401/da6401 assignment 2/wandb/run-20250418_204426-24f1so8s/logs/debug-internal.log
2025-04-18 20:44:26,839 INFO    Thread-2 (_run_job):68003 [wandb_init.py:init():761] calling init triggers
2025-04-18 20:44:26,839 INFO    Thread-2 (_run_job):68003 [wandb_init.py:init():766] wandb.init called with sweep_config: {'augment': False, 'batch_norm': True, 'batch_size': 128, 'conv_activation': 'sigmoid', 'dense': [256], 'dense_activation': 'silu', 'dropout': 0.1, 'filter_initial': 64, 'filter_strategy': 'halved', 'kernel': [5, 5, 5, 5, 5], 'learning_rate': 0.0006321850009384904, 'momentum': 0.9782965971049628, 'optimizer': 'adamw', 'resize': 64, 'weight_decay': 0.008237184098140907}
config: {'batch_size': 32, 'resize': 128, 'filter_automated': True, 'filter_strategy': 'doubled', 'filter_initial': 32, 'n_convolutions': 5, 'filter_manual': [16, 32, 64, 128, 256], 'padding': True, 'stride': [1, 1, 1, 1, 1], 'dense': [64], 'kernel': [3, 3, 3, 3, 3], 'conv_activation': 'relu', 'dense_activation': 'relu', 'n_dense': 1, 'optimizer': 'adam', 'augment': False, 'batch_norm': False, 'learning_rate': 0.001, 'momentum': 0.9, 'weight_decay': 0, 'dropout': 0, 'xavier_init': False, 'wandb': False, 'wandb_entity': 'da24d008-iit-madras', 'wandb_project': 'da6401-assignment2', 'wandb_sweep': True, 'sweep_id': None, '_wandb': {}}
2025-04-18 20:44:26,839 INFO    Thread-2 (_run_job):68003 [wandb_init.py:init():784] starting backend
2025-04-18 20:44:26,839 INFO    Thread-2 (_run_job):68003 [wandb_init.py:init():788] sending inform_init request
2025-04-18 20:44:27,185 INFO    Thread-2 (_run_job):68003 [backend.py:_multiprocessing_setup():97] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2025-04-18 20:44:27,186 INFO    Thread-2 (_run_job):68003 [wandb_init.py:init():803] backend started and connected
2025-04-18 20:44:27,187 INFO    Thread-2 (_run_job):68003 [wandb_run.py:_config_callback():1261] config_cb None None {'augment': False, 'batch_norm': True, 'batch_size': 128, 'conv_activation': 'sigmoid', 'dense': [256], 'dense_activation': 'silu', 'dropout': 0.1, 'filter_initial': 64, 'filter_strategy': 'halved', 'kernel': [5, 5, 5, 5, 5], 'learning_rate': 0.0006321850009384904, 'momentum': 0.9782965971049628, 'optimizer': 'adamw', 'resize': 64, 'weight_decay': 0.008237184098140907}
2025-04-18 20:44:27,189 INFO    Thread-2 (_run_job):68003 [wandb_init.py:init():896] updated telemetry
2025-04-18 20:44:27,217 INFO    Thread-2 (_run_job):68003 [wandb_init.py:init():920] communicating run to backend with 90.0 second timeout
2025-04-18 20:44:27,837 INFO    Thread-2 (_run_job):68003 [wandb_init.py:init():995] starting run threads in backend
2025-04-18 20:44:28,044 INFO    Thread-2 (_run_job):68003 [wandb_run.py:_console_start():2377] atexit reg
2025-04-18 20:44:28,044 INFO    Thread-2 (_run_job):68003 [wandb_run.py:_redirect():2227] redirect: wrap_raw
2025-04-18 20:44:28,044 INFO    Thread-2 (_run_job):68003 [wandb_run.py:_redirect():2292] Wrapping output streams.
2025-04-18 20:44:28,044 INFO    Thread-2 (_run_job):68003 [wandb_run.py:_redirect():2317] Redirects installed.
2025-04-18 20:44:28,047 INFO    Thread-2 (_run_job):68003 [wandb_init.py:init():1037] run started, returning control to user process
2025-04-18 20:45:09,538 INFO    MainThread:68003 [wandb_run.py:_finish():2112] finishing run da24d008-iit-madras/da6401-assignment2/24f1so8s
2025-04-18 20:45:09,539 INFO    MainThread:68003 [wandb_run.py:_atexit_cleanup():2342] got exitcode: 0
2025-04-18 20:45:09,539 INFO    MainThread:68003 [wandb_run.py:_restore():2324] restore
2025-04-18 20:45:09,540 INFO    MainThread:68003 [wandb_run.py:_restore():2330] restore done
2025-04-18 20:45:10,140 WARNING MsgRouterThr:68003 [router.py:message_loop():77] message_loop has been closed
Traceback (most recent call last):
  File "/opt/anaconda3/envs/nlp_env/lib/python3.12/site-packages/wandb/sdk/interface/router_sock.py", line 28, in _read_message
    resp = self._sock_client.read_server_response(timeout=1)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/anaconda3/envs/nlp_env/lib/python3.12/site-packages/wandb/sdk/lib/sock_client.py", line 285, in read_server_response
    data = self._read_packet_bytes(timeout=timeout)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/anaconda3/envs/nlp_env/lib/python3.12/site-packages/wandb/sdk/lib/sock_client.py", line 270, in _read_packet_bytes
    raise SockClientClosedError
wandb.sdk.lib.sock_client.SockClientClosedError

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/opt/anaconda3/envs/nlp_env/lib/python3.12/site-packages/wandb/sdk/interface/router.py", line 70, in message_loop
    msg = self._read_message()
          ^^^^^^^^^^^^^^^^^^^^
  File "/opt/anaconda3/envs/nlp_env/lib/python3.12/site-packages/wandb/sdk/interface/router_sock.py", line 30, in _read_message
    raise MessageRouterClosedError from e
wandb.sdk.interface.router.MessageRouterClosedError
2025-04-18 20:45:10,142 INFO    MsgRouterThr:68003 [mailbox.py:close():115] Closing mailbox, abandoning 2 handles.
